src/salesforce_uploader.js

```diff
@@ -0,0 +1,63 @@
+import { fetchConfig } from './lib/config'
+import AWS from 'aws-sdk'
+import NamedError from './lib/NamedError'
+import { authenticate } from './lib/salesforceAuthenticator'
+
+const s3 = new AWS.S3({ signatureVersion: 'v4' })
+const BUCKET = 'fulfilment-output-test'
+
+export function handler (input: mixed, context, callback) {
+  if (!(input.path && typeof input.path === 'string')) {
+    callback(new NamedError('inputError', 'Input did not contain filename'))
+    return
+  }
+  uploader(input).then((r) => {
+    console.log(r)
+    console.log('success')
+    callback(null, r)
+  }).catch(e => {
+    console.log('oh no  ')
+    console.log(e)
+    callback(e)
+  })
+}
+
+async function uploader (input: { path: string }) {
+  const config = await fetchConfig()
+  const salesforce = await authenticate(config)
+  console.log('Finding fulfilment folder.')
+  const folder = await salesforce.getFulfilmentFolder()
+  console.log(folder)
+  // get file from s3 as stream
+
+  let options = { Bucket: BUCKET, Key: `${config.stage}/${input.path}` }
+  console.log(`Retreiving file ${options.Key} from S3 bucket ${options.Bucket}.`)
+  let d = await s3.getObject(options).promise()
+
+  // build a little json
+  let message = {
+    'Description': 'a test file',
+    'Keywords': 'hello!',
+    'FolderId': folder,
+    'Name': input.path,
+    'Type': 'csv'
+  }
```

Comment:
are these the correct description and keywords we want for this ? or does it not matter at all?